docs/api/materials/ShaderMaterial.html

```diff
@@ -31,11 +31,11 @@
 		var material = new THREE.ShaderMaterial( {
 
 			uniforms: {
-				time: { type: "f", value: 1.0 },
```

Comment:
I guess while we're at it, we can change this to be...

```js
uniforms: {
    time: 1.0,
    resolution: new THREE.Vector2()
}
```

Just, internally, we'll have to check if `uniform.value` exists and use that instead.

What do you think?